src/Model/EndpointLocator.php

```diff
@@ -35,12 +36,11 @@ public function set(string $alias, RepositoryInterface $repository): Endpoint
      *
      * @param string $alias The alias name you want to get.
      * @param array $options The options you want to build the endpoint with.
-     * @return \Muffin\Webservice\Model\Endpoint
+     * @return \Cake\Datasource\RepositoryInterface
      * @throws \RuntimeException If the registry alias is already in use.
      */
-    public function get(string $alias, array $options = []): Endpoint
+    public function get(string $alias, array $options = []): RepositoryInterface
```

Comment:
```suggestion
    public function get(string $alias, array $options = []): Endpoint
```

Endpoint locator must always return an Endpoint :)

Reply:
The interface doesn't doesn't require that, but it makes sense indeed. Changed.